### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by ellensig**

### CODE REVIEW

File: components/DashboardContainer/index.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
     <div className={styles.dashContainer}>
       <Label styler='labelDashContainer' text='Favorites' htmlFor='dashboardContainer' />
       <Dashboard />
-      <Button title='See all' styler='seeAll' />
+      <div className={styles.dashboardFooter}>
+        <NextLink href='/seealldash'>
+          <a> <Image size='seeAllIcon' url='/static/seeall.svg' alt='see all icon' /> </a>
+        </NextLink>
+        <NextLink href='/seealldash'>
+          <a className={styles.seeAllLink}> See All </a>
```
Comment (audunnhrafn) [inline components/DashboardContainer/index.js]: this does not need to be <a>

File: components/DashboardContainerMand/assets/index.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.dashContainer {
+  // background-color: honeydew;
+  margin-top: 20px;
+  // margin-left: 23%;
+  margin-left: 11%;
+  width: 77%;
+  // margin-left: 320px;
+  // width: 2300px;
```
Comment (audunnhrafn) [inline components/DashboardContainerMand/assets/index.scss]: comments

File: components/DashboardMand/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <div className='dashboard'>
+      <div className={styles.dashboardModal}>
+        <DashboardItemMand />
+      </div>
+    </div>
+  )
+}
+
+export default Dashboard
```
Comment (audunnhrafn) [inline components/DashboardMand/index.js]: unused component

File: components/DashboardContainer/index.js (hunk 1 above)
Comment (ellensig) [reply components/DashboardContainer/index.js]: Yes it does

**Pull request merged**